scripts/python/missingBCC.py

```diff
@@ -49,8 +51,12 @@ def countFile(dir, filename):
                     totalBCC += 1
                     totalSites += 1
                 processed = True
+        if pathFail:
+            global failedPaths 
+            failedPaths += 1
         index += 1
 
+
 counter = 0
 while counter < len(filename):
     path = dirPath + "/" + filename[counter]
```

Comment:
Actually, all this path stuff is os specific.  Not sure it's worth fixing.

Reply:
I could fix it shouldn't take long anyways